syntax/problem.php

```diff
@@ -147,18 +164,69 @@ private function renderFigures(Doku_Renderer &$renderer, \PluginFKSTaskRepo\Task
         }
     }
 
+    /**
+     * Outputs no image files
+     * @param Doku_Renderer $renderer
+     * @param \PluginFKSTaskRepo\Task $data
+     */
+    private function renderFileAttachments(Doku_Renderer &$renderer, \PluginFKSTaskRepo\Task $data) {
+        if (is_array($data->getFigures())) {
+            $renderer->doc .= '<div class="task-fileattachments">';
+            foreach ($data->getFigures() as $figure) {
+                if($this->isImage(ml($figure['path']))) continue; // Checks if it is an image
+                $renderer->doc .= '<div class="task-fileattachments-file">';
+                $renderer->internalmedia($figure['path'], $figure['caption'] ?: null,null,null,null,null,'linkonly');
+                $renderer->doc .= '</div>';
+            }
+            $renderer->doc .= '</div>';
+        }
+    }
+
     private function renderTags(Doku_Renderer &$renderer, \PluginFKSTaskRepo\Task $data) {
         $tags = $this->helper->loadTags($data->getYear(), $data->getSeries(), $data->getLabel());
         foreach ($tags as $tag) {
             $renderer->doc .= $this->helper->getTagLink($tag, null, $data->getLang());
         }
     }
 
+    private function renderOrigin(Doku_Renderer &$renderer, \PluginFKSTaskRepo\Task $data) {
+        $renderer->doc .= '<div class="origin">' . $renderer->render_text($data->getOrigin()) . '</div>';
```

Comment:
použiť namiesto css, už predpripravené boostrap css font-italic a pull-right (float-right)

Reply:
To nechceš, není výslovně řečeno, že by to mělo být vpravo, nebo kurzívou.

Reply:
style.less L80?

Reply:
😑 ale tak to jsem si řekl jen já a kdokoliv to může změnit, udělat to červené, dát to doprostřed atd...

Reply:
Tak to opravíš tu a nie v CSS...

Myslíš že Výfuk to bude chcieť tiež takto? Od toho sú renderery aby sa to rozlišovalo. A predovšetkým načo písať niečo čo už je v bootstrape? je to o rád prehľadnejšie keď uvidím triedu pull-right hneď viem čo to robí, takto to musím dohľadávať v CSSkach